jestconfig.json

```diff
@@ -7,5 +7,17 @@
   "testEnvironment": "jsdom",
   "setupFilesAfterEnv": [
     "./src/setupTests.ts"
+  ],
+  "coverageThreshold": {
+    "global": {
+      "statements": 0,
+      "branches": 0,
+      "functions": 0,
+      "lines": 0
+    }
+  },
+  "reporters": [
+    "default",
+    ["jest-junit", { "outputDirectory": "./test-results" }]
```

Comment:
What is this tool for? just curiosity

Reply:
It is a plugin for jest, that allows generating XML reports. I read that this may be helpful for CI/CD tools

Reply:
BTW, if we don't need it, I can remove it.